src/services/cosmos.ts

```diff
@@ -472,4 +473,61 @@ export class Cosmos {
           .map(r => StakingMetadata.remove(r));
       });
   }
+
+  async syncGovernanceVoting() {
+    try {
+      const rows = realm.objects<GovernanceVoting>(
+        GovernanceVoting.schema.name,
+      );
+      const cache: string[] = [];
+
+      for (const row of rows) {
+        cache.push(row.hash);
+      }
+
+      const proposals = await this.getProposals();
+      const hashes = proposals.proposals
+        .map(
+          ({
+            status,
+            voting_end_time,
+            voting_start_time,
+            total_deposit,
+            deposit_end_time,
+            proposal_id,
+            final_tally_result,
+            content,
+          }) => {
+            const veto = final_tally_result.no_with_veto;
+
+            const copy: any = {...final_tally_result};
+            delete copy.no_with_veto;
+            const votes: any = {};
+
+            Object.entries({...copy, veto}).map(([key, val]) => {
+              votes[key] = Math.round(Number(val));
+            });
```

Comment:
pass in GovernanceVoting.createVoting whole object Proposal and perform data in there